src/item_factory.h

```diff
@@ -266,6 +266,13 @@ class Item_factory
 
         /** Find all item templates (both static and runtime) matching UnaryPredicate function */
         static std::vector<const itype *> find( const std::function<bool( const itype & )> &func );
+        /**
+         * Find all holsters for itype. Uses cache, so it's much faster than doing on the fly.
```

Comment:
It would be a bit clearer if you mentioned the specific cache (`type_contained_in_holsters`) rather than just that a cache is used.

Reply:
done